### DISCUSSION TITLE
nav links to markdown files in subdirectories

### DISCUSSION THREAD

**Discussion by madmarcos-codeup** [mkdocs/mkdocs#3054] [Q&A] [answered]
Hi there,
Pardon my ignorance, but I am very new to mkdocs.
I have a nav section in my mkdocs.yml file with MD files that are in a variety of subdirectories.
For example, the nav section in my mkdocs.yml file looks similar to this:
```
nav:
-  Topic 1
  - Item A: item_a.md
  - Item B: item_b.md  
-  Topic 2
  - Item C: subdir1/item_c.md
  - Item D: subdir1/item_d.md
```

In the resulting HTML nav bar, the links to Items A and B are correct. But, the links to Items C and D are incorrect and look like `https://my.example.com/subdir1/subdir1/item_c.html`

In the above example, the item_c HTML file is created as `subdir1/item_c.html`. So, I would expect the nav link URL for item_c to be `https://my.example.com/subdir1/item_c.html`

I am just using `mkdocs build` to build the HTML.

Thank you for any help you can provide!

**Comment by oprypin** [comment, marked as answer]
According to everything that I know about MkDocs, there's no way to get this `subdir1/subdir1` out of nowhere if you didn't do something to make it so. So I think you might be missing some step you took and not describing it here. For that reason, I would ask you to share a complete directory with an MkDocs site that exhibits this behavior, then maybe we'll find something.

**Reply by madmarcos-codeup** [reply]
Thanks for the response. I started over with a new project and the nav behavior is correct. As you suggested, something is wrong with the configuration of my inherited project. I will take its config apart and try to isolate what is causing the problem. 
Thanks again!

**Reply by madmarcos-codeup** [reply]
the problem was with the outdated theme that was being used.
I have fixed the problem. 
Thank you!
